pkgs/applications/science/biology/sratoolkit/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv, lib
+, fetchurl
+, autoPatchelfHook
+, libidn
+, zlib
+, bzip2
+}:
+
+stdenv.mkDerivation rec {
+  name = "sratoolkit";
+
+  version = "2.11.3";
+
+  libidn11 = libidn.overrideAttrs (old: {
```

Comment:
Do we really need this? Can we patch instead? If not then please move this at least in a let in. Also please use pname+version.

Reply:
Could you please elaborate what you mean with can we patch instead? We cannot use a newer version without patching somehow a binary.

Reply:
I thought we would be building from source

Reply:
The effort to build this from the sources is disproportionate.

Reply:
What makes it so hard?

Reply:
For building, we would need ncbi-vdb and ngs. ncbi-vdb and ngs depend on parts on each other. Both require a couple of libraries (~20, from my mind, could be wrong). Many are not in nixpkgs. I have put more than 8h to building this, and failed on more than basic libraries.